Easter Eggs

Walkercito opened the pull request:
- Added an easter egg: if the user tries to drop the `users` table (`DROP TABLE users;`), a Windows XP–inspired BSoD will appear for a few seconds.
- Implemented proper handling for empty query input. Without this check, the console would throw an error.
- Added `await` to the processor function call on line 254.
- Added a button in the Auth modal to enable/disable the CRT effect.

Walkercito committed:
feat: add drop easter egg

Added DROP TABLE user; Easter Egg, fixed un-awaited function on line 245 and added proper proper handler for when the query is empty

Walkercito committed:
add: ctr on/off button

Quick add of the CRT enable and disable button in the Auth modal

A5rocks commented on src/auth_modal.py:
```diff
@@ -20,20 +21,27 @@
     AUTH_FORM = document.getElementById("auth-form")
     STATUS_TEXT = document.getElementById("security-notice")
 
+    # Debug logging
```
debug logging should be removed

Walkercito replied:
okay

A5rocks reviewed (approved):
Assuming this works, this looks good

Walkercito committed:
edit: bug fixes and debug

Fix: Handle stealth mode and missing post structures

- Fix KeyError when extracting images from non-post data (suggestions, followers)
- Fix AttributeError when getting own profile in stealth mode
- Add friendly user messages instead of Python exceptions
- Improve error handling for anonymous API access

Walkercito commented:
Well, my last commit: 
- Fix KeyError when extracting images from non-post data (suggestions, followers)
- Fix AttributeError when getting own profile in stealth mode  
- Add friendly user messages instead of Python exceptions
- Improve error handling for stealth API access
- Removed debug logging

Walkercito committed:
edit: better post handling

Walkercito committed:
fix: profile fix

Walkercito committed:
fix: where clause fix

Walkercito committed:
fix: lint errors

Walkercito committed:
edit: typo of mine

DannyTheHeretic committed:
fix: no more console.log plain text passwords
